screens/Auth/Login.js

```diff
@@ -16,7 +16,7 @@ export default class SignInScreen extends Component {
 
     goToMain = (userObj) => {
       const navigateAction = NavigationActions.navigate({
-        routeName: "Home",
+        routeName: "Words",
```

Comment:
I'm guessing this was for testing purposes, but it should be switched back to `Home`.

Reply:
It's not for testing purposes. Its the new route to Words

Reply:
Ahh, because that's the page we'll see instead of a homepage. Got it! 👍